assets/components/src/action-card/style.scss

```diff
@@ -120,4 +120,18 @@
 	&:hover button.newspack-action-card__secondary_button {
 		visibility: visible;
 	}
+
+	/* Badge */
+
+	.newspack-action-card-badge {
+		border: 1px solid #606A73;
+		border-radius: 3px;
+		color: #606A73;
```

Comment:
Not sure if this is something we have access to like the Muriel colors, but if so this color is available through Gutenberg colors: https://github.com/Automattic/jetpack/blob/22297f2ecbc43ebc425dadb1e83f025ca4daff7e/extensions/shared/styles/gutenberg-colors.scss#L16

Reply:
After discussing with @sonjaleix changed to Muriel Gray 500 in https://github.com/Automattic/newspack-plugin/pull/203/commits/443e11df7d863fcbfdf74c54b3b749a535c974cc